portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,58 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Manuela Rodriguez</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+      <h1>My Portfolio: Manuela Rodriguez</h1>
+      <p>My name is Manuela Rodriguez, I'm a third-year student at New York University studying Computer Science and Business 
+          with a concentration in Finance. I am passionate about using data and technology to solve complex and ambiguous problems,
+          and making Computer Science more accessible to historically underrepresented communities.
+      </p>
+      <p> <img src="/images/headshot.JPG" width="467.8" height="311.9"/> </p>
+      <p>Click here to get a random fact about me:</p>
+      <button onclick="addRandomFact()">Get a fact!</button>
+      <div id="fact-container"></div>
+
+      <h3>Check out my LinkedIn and Github profiles!</h3>
+      <ul> 
+      <li><a href="https://www.linkedin.com/in/manuela-rodriguez-a8a3b4157/">LinkedIn</a></li>
+      <li><a href="https://github.com/manuelarod">Github</a></li>
+      </ul>
+
+      <h2> Projects </h2>
+      <h4> National Museum of American Diplomacy ImpactHack 2020</h4> 
+      <ul>
+          <li> Worked in a team of four to build a full-stack web application where users can take an 
+                interactive quiz that matches their skills and interests to careers in diplomacy. </li>
+                <li> Designed an efficient data structure and ranking algorithm to match the user’s responses 
+                  with career tracks in diplomacy.</li>
+            <li><a href="https://hidden-shore-16633.herokuapp.com/">See project</a></li>
+      </ul>
+      <h4> The Restaurant Industry Amidst the COVID-19 Pandemic</h4>  
+      <ul>
+                <li> Used publicly available data to provide stakeholders 
```

Comment:
It looks like some of these lines (the <li> ones here) got pushed up an extra tab. Try copying this file into:

https://www.freeformatter.com/html-formatter.html

and it'll format it automatically for you!